<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueViewClass</class>
 <widget class="QWidget" name="QueueViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="queueInfoPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>330</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="queueAnimAreaPane" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>110</y>
     <width>691</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QGraphicsView" name="queueAnimArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>691</width>
     <height>97</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="queueHeaderAndControlPane">
    <item>
     <widget class="QLabel" name="queueHeaderLabel">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;QUEUE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="queueControlPane">
      <item>
       <widget class="QLineEdit" name="queueInsertLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="queueEnqBtn">
        <property name="text">
         <string>Enqueue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="queueEnqDeqSepLine">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="queueDeqBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Dequeue</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="queueHomeBtn">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="queueInfoConsoleLabel">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>310</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Theory:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="queuePlainTextEdit">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>130</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="queueConsoleLabel">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>110</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Console:</string>
   </property>
  </widget>
  <widget class="Line" name="queueAnimTextAreaSeparator">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>31</width>
     <height>411</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
